docs/topics/eap.md

```diff
@@ -32,6 +32,19 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>Recommended kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.6.20-RC</strong>
+            <p>Released: <strong>February 21, 2022</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>Stabilization of Kotlin 1.6.20-M1 features.</p>
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC">changelog</a>.</p>
+        </td>
+        <td>
+            <p>The same as for the Kotlin 1.6.20-M1, see the row below.</p>
```

Comment:
Let's maybe just merge cells?

Reply:
Ah, this looks really awkward with our table style. Forget it.
Just remove the first two "the" articles in your sentence.